signbank/dictionary/tasks.py

```diff
@@ -91,9 +92,10 @@ def retrieve_videos_for_glosses(video_details: List[VideoDetail]):
             is_public=False,
             video_type=video_type
         )
-        videos_to_create.append(
-            move_glossvideo_to_valid_filepath(gloss_video)
-        )
+        if settings.DEBUG:
+            gloss_video = move_glossvideo_to_valid_filepath(gloss_video)
```

Comment:
**question**: is this really linked to whether the application is in `DEBUG` mode? We might want to use a variable if the actual thing being checked is something else & we're just checking if we're using development settings

Reply:
I'll change it to check which GLOSS_VIDEO_FILE_STORAGE it's using, it's probably the more appropriate check